sale_credit_point/tests/test_credit_history.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests.common import TransactionCase
+from odoo import fields
+
+
+
+class TestHistory(TransactionCase):
+
+    def setUp(self):
+        super().setUp()
+        self.history_model = self.env["credit.point.history"]
+        self.partner = self.env['res.partner'].create({
```

Comment:
`.with_context(tracking_disable=True)` I know, I missed it in my tests, add it there too pls :wink: